HW-0.Rmd

```diff
@@ -9,6 +9,6 @@ Your tasks!
 
 * Change the name of the author above to yours
 * Change the date
-* Replace the NHL URL with the URL for the document you just posted to Rpubs **here**: [My RPubs Document](https://www.nhl.com/)
+* Replace the NHL URL with the URL for the document you just posted to Rpubs **here**: [My RPubs Document](http://rpubs.com/dixonjacob/TURINGKNIT)
 * Click Knit HTML
-* Please indicate how much time you spent on this assignment: 
+* Please indicate how much time you spent on this assignment: 30 min
```

Comment:
Everything looks good here. Rpubs is a useful tool for quickly publishing text, R code/output to the web, so you can share it with collaborators.